fryr/demo.html

```diff
@@ -0,0 +1,75 @@
+---
+layout: page
+title: Fryr Demo
+repository_slug: fryr
+js_download: true
+permalink: /fryr/demo
+tagline: Command window.location.hash like a cook in the kitchen
+logo: fryr.svg
+show_docs: true
+---
+<form class="wrapper">
+  <div class="input">
+    <label for="key">Key</label>
+    <input type="text" id="js-fryr-key" name="key" value="office_furniture" placeholder="Param value" />
+  </div>
+  <div class="input">
+    <label for="value">Value</label>
+    <input type="text" id="js-fryr-value" name="value" value="aquarium" placeholder="Param key" />
+  </div>
+
+  <section class="download">
+    <input type="submit" id="js-fryr-update" class="button" value="update()">
+    <a href="#" id="js-fryr-append" class="button">append()</a>
+    <a href="#" id="js-fryr-reset" class="button">Reset</a>
+  </section>
+</form>
+
+<pre id="js-params"></pre>
+
+<script type="text/javascript" src="./fryr.js"></script>
```

Comment:
Instead of adding another version of fryr.js to keep up-to-date, what do you think about just linking it to https://raw.githubusercontent.com/wearefine/fryr/master/fryr.js? Wouldn't normally do that in a client project but might make sense here.